core/src/builder.ts

```diff
@@ -77,13 +86,27 @@ function PropertyBuilder<Model>(initialProps?: Partial<Model>) {
 function Builder<Model>({
   generator,
   transformers,
+  type,
+  postBuild,
+  name = 'Unknown Builder',
+  compatConfig,
```

Comment:
This is for the compatibility mode of the new pattern where a single builder needs to support both representations.